website/versions/source-awspricing.json

```diff
@@ -0,0 +1 @@
+{ "latest": "plugins-source-awspricing-v1.0.0" }
```

Comment:
No need for this file. Without it the plugin will show as "Unpublished" until the version is released which I think is better than showing a version that doesn't exist yet